assemblies/assembly-customize-rhdh-theme.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="assembly-customize-rhdh-theme"]
+= Customzing the appearance of your {product} instance
+:context: assembly-customize-rhdh-theme
+
+There are two default theme configurations available for {product}:
```

Comment:
I think if we are going to leave "two" here, we don't want to also say "include". Otherwise we should say "The default theme configurations include the following:" Or "The following configurations are for the default themes available for {product}: